docs/how-to.md

```diff
@@ -7,12 +7,12 @@
 
 ## Code in Master
 
-Install the dependencies
+Install the dependencies:
 ```bash
 $ npm install
 ```
 
-Run the tests :heavy_check_mark:
+Run the tests: :heavy_check_mark:
```

Comment:
Maybe we should remove the check mark then. It seems weird after a colon.

Reply:
Ok, I will remove it 👍